competitions/MICCAI/surgtoolloc/classification_files/cfg_efnb4.yaml

```diff
@@ -0,0 +1,62 @@
+---
+imports:
+- "$import types"
+cfg:
+    _target_: types.SimpleNamespace
+    output_dir: "./output/"
+    data_dir: "/raid/surg/image640_blur/"
+    backbone: "efficientnet-b4"
+    train_df: "cleaned_clf_train_data.csv"
+    img_size: [640, 640]
+    batch_size: 196
+    num_classes: 14
+    lr: 0.001
+    epochs: 5
+    oversample_rate: 4
+    clf_threshold: 0.4
+    num_workers: 8
+    gpu: 0
+    device: "cuda:0"
+    train_aug:
+        _target_: Compose
+        transforms:
+            - _target_: LoadImaged
+              keys: "input"
+              image_only: true
```

Comment:
thanks @yiheng-wang-nv, do you think this could be simplified like this preprocssing:

https://github.com/Project-MONAI/model-zoo/blob/0ddf219c7ce9a2a313450a5633ba0f02a033c6c1/models/mednist_reg/configs/inference.yaml#L14-L42

train_aug could use `$@image_load + @image_aug` and val_aug uses `@image_load`. so this ensures consistent training vs validation